src/main/java/com/dev/bootbasic/view/Command.java

```diff
@@ -0,0 +1,19 @@
+package com.dev.bootbasic.view;
+
+public enum Command {
+
+    CREATE,
+    LIST,
+    EXIT;
+
```

Comment:
[super nit]
```suggestion

    CREATE,
    LIST,
    EXIT,
    ;

```
아주 nit한 거지만 요렇게 해주시면 나중에 새로운 값이 추가되거나 지워졌을 때 git에서 체크하는 변경 사항이 실제 변경된 부분만 볼 수 있어서 좋습니다!

Reply:
오.. 꿀팁 감사합니다! 반영했습니다!
[d599980](https://github.com/prgrms-be-devcourse/springboot-basic/pull/694/commits/d599980dbb74530f7244490ce2501ce4aca756b0)